### ISSUE TITLE
Concern on Admin User - Prompts within Re-approving Reject reservations. Add prompt boxes as warnings before commiting re-approval attempts. 

### ISSUE THREAD

**Issue by ralch34** [philrhoy/IMCCVenueReservationSystem2023#32]
Concern on Calendar Display: Rejected Reservations must not be displayed on the Calendar, 
only the Pending and the Approved Reservations...

In cases of Rejected Reservations given a 2nd chance (after edits) if DSA changes its status to "approved". 
THE RESERVATION MAY DISPLAY AGAIN IF EVER DILI SIYA MAGCONFLICT WITH OTHER ALREADY APPROVED RESERVATION SCHED.

_This leads use to another issue, "When attempting to create two reservations with the same venue, date, and time range, an error code pops up."_ details will be mention in this issue later...

Let's go back on topic, to avoid other issues that may arise, you put a WARNING PROMPT FOR DSA Admin user when attempting to change a Rejected Reservation into an Approved Reservation. 

After apply changes to the status,

a Prompt box appears....  
example no .1:
"Warnings: You are attempting to approve this Rejected Reservation schedule that conflicts with already approved reservation RES002823, Conflict detected!" 
(System Displays RES002823 Venue and Sched Details)    
"You cannot proceed unless assign this reservation to a different venue, date, or time range"
"Back to Edit" nga button.
_________
example no. 2: 
"Warnings: You are attempting to approve this Rejected Reservation RES003232, no schedule conflicts detected, no missing uploads detected"  
"Are you sure you want to Re-approve this Rejected Reservation"
Buttons nga "Yes/Proceed" and "No"

___________
example no. 3:
"Warnings: There is no Uploaded Student Activity Form in this reservation., missing uploads detected!"
"You cannot proceed unless upload the supporting documents for this reservation."
"Back to Edit" nga button.

____________
example no. 4:
"Warnings: There is no Uploaded Letter of Approval in this reservation, missing uploads detected!"
"Are you sure you want to Re-approve this Rejected Reservation"
"Back to Edit" nga button.   or "Yes/Proceed" nga button

This is the case naay times considered ra walay Letter of Approval, as long as naa ang uploaded Student Activity Form.
Magprovide gihapon tag Prompt... gihapon...
___________
IF SUCCESSFUL ANG RE-APPROVAL SA RESERVATION, The Reservation must re-appear again in the Calendar.
____

(this pic below shows I submitted Three new reservations, each with different statuses)
![Screenshot_34 after gale ma reject sa admin ang reservation remove it in the calendar](https://github.com/philrhoy/IMCCVenueReservationSystem2023/assets/90671718/2b90d517-10cf-43df-a49a-19dad79562e4)

(What displays on the calendar after those three reservations)
![Screenshot_35 after gale ma reject sa admin ang reservation remove it in the calendar](https://github.com/philrhoy/IMCCVenueReservationSystem2023/assets/90671718/e76d1910-e720-4972-a15c-6d37ec280ba0)
